runner/runner/celery.py

```diff
@@ -38,12 +36,27 @@
     },
 }
 
+
 app = Celery(
     "runner",
-    broker=f"redis://{REDIS_HOST}:{REDIS_PORT}",
     include=["runner.handlers"],
 )
 
+app.conf.update(
+    {
+        "broker_url": "filesystem://",
+        "broker_transport_options": {
+            "data_folder_in": "/var/lib/broker/data",
+            "data_folder_out": "/var/lib/broker/data",
+            "data_folder_processed": "/var/lib/broker/processed",
```

Comment:
If I understood correctly, I just removed the `data_folder_processed` option, and now the `data_folder_in/out` just points to the root of the `broker` directory.

Reply:
Yeah, what you did is fine.  I was originally suggesting that you still have a `/data` subdirectory so that you don't have to change the naming if we later decide that we need that `/processed` directory.  Basically, if the in and out folder is just `/tmp/broker` and we later want to add a processed directory, what is the path to that?  `/tmp/broker/data` makes the answer to that question obvious: `/tmp/broker/processed`.  I don't feel super strongly about it though and am fine with what you went with.